.storybook/styles/components/_forms.scss

```diff
@@ -1,7 +1,8 @@
 /*-----------------------
-Fieldset
+Field Wrappers
 -----------------------*/
-fieldset {
+fieldset,
+.field-wrapper {
```

Comment:
My hope is that this might ease some of the style changes (just swapping `fieldset` for `.field-wrapper` in stylesheets). Hopefully most projects can just find and replace in their stylesheets without too much auditing.

Reply:
Can you add this documentation to a migration guide?